.github/workflows/gen-whoami-table.yml

```diff
@@ -3,37 +3,45 @@ name: "Regenerate WhoAmI table"
 
 on:
   workflow_dispatch:
+  pull_request:
```

Comment:
Ye this makes sense. Otherwise the action may fail after merging lol

Reply:
Yup, ideally all CI-type workflows which run on `push`, `pull_request`, and without a branch filter. Anything else tends to delay finding problems which kind-of defeats the point.